module/VuFind/src/VuFind/Db/Row/UserList.php

```diff
@@ -305,4 +308,118 @@ public function delete($user = false, $force = false)
         // Remove the list itself:
         return parent::delete();
     }
+
+    /**
+     * Get identifier (returns null for an uninitialized or non-persisted object).
+     *
+     * @return ?int
+     */
+    public function getId(): ?int
+    {
+        return $this->id ?? null;
+    }
+
+    /**
+     * Title setter
+     *
+     * @param string $title Title
+     *
+     * @return UserListEntityInterface
+     */
+    public function setTitle(string $title): UserListEntityInterface
+    {
+        $this->title = $title;
+        return $this;
+    }
+
+    /**
+     * Get Title.
+     *
+     * @return string
+     */
+    public function getTitle(): string
+    {
+        return $this->title;
+    }
+
+    /**
+     * Description setter
+     *
+     * @param string $description Description
+     *
+     * @return UserListEntityInterface
+     */
+    public function setDescription(string $description): UserListEntityInterface
+    {
+        $this->description = $description;
+        return $this;
+    }
+
+    /**
+     * Get Description.
+     *
+     * @return string
+     */
+    public function getDescription(): string
+    {
+        return $this->description;
+    }
+
+    /**
+     * Created date setter.
+     *
+     * @param DateTime $dateTime Created date
+     *
+     * @return UserListEntityInterface
+     */
+    public function setCreated(DateTime $dateTime): UserListEntityInterface
+    {
+        return $this->created = $dateTime;
+    }
+
+    /**
+     * Get created date.
+     *
+     * @return DateTime
+     */
+    public function getCreated(): DateTime
+    {
+        return $this->created;
+    }
+
+    /**
+     * Set the list public.
+     *
+     * @param bool $public Created date
+     *
+     * @return UserListEntityInterface
+     */
+    public function setIsPublic(bool $public): UserListEntityInterface
+    {
+        $this->public = $public ? 1 : 0;
+        return $this;
+    }
+
+    /**
+     * Check if the list is public
+     *
+     * @return bool
+     */
+    public function getPublic(): bool
+    {
+        return (bool)$this->public ?? false;
+    }
+
+    /**
+     * Set user ID.
+     *
+     * @param ?UserEntityInterface $user User owning token
+     *
+     * @return UserListEntityInterface
+     */
+    public function setUser(?UserEntityInterface $user): UserListEntityInterface
```

Comment:
You seem to be missing the parallel `getUser` method. You can borrow the implementation from here:

https://github.com/vufind-org/vufind/pull/3650/files#diff-f816a494120067ba2992dab2a717b2c7b6410ca6728af31a2cdb89c451d7098bR259-R269

Reply:
I see you've added getUser below, but it's just returning the user_id, and it needs to return the full object. See the link above for an example of how to do this.